ADAL/src/ADAuthenticationContext.m

```diff
@@ -397,6 +397,21 @@ - (void)acquireTokenWithResource:(NSString *)resource
     [request acquireToken:@"133" completionBlock:completionBlock];
 }
 
+- (void)acquireTokenWithRefreshToken:(NSString *)refreshToken
+                            resource:(NSString *)resource
+                            clientId:(NSString *)clientId
+                         redirectUri:(NSURL *)redirectUri
+                     completionBlock:(ADAuthenticationCallback)completionBlock
+{
+    API_ENTRY;
+    REQUEST_WITH_REDIRECT_URL(redirectUri, clientId, resource);
+    CHECK_STRING_ARG_BLOCK(refreshToken);
+    [request setRefreshToken:refreshToken];
+    [request setSilent:YES];
+    
+    [request acquireToken:@"136" completionBlock:completionBlock];
```

Comment:
i used a new API id 136 here, probably the telemetry doc should be updated as well? :)

Reply:
yes